src/subscription_manager/utils.py

```diff
@@ -232,12 +248,13 @@ def get_version(versions, package_name):
     return "%s%s" % (package_version, package_release)
 
 
-def get_terminal_width():
+def get_terminal_width() -> Optional[int]:
     """
     Attempt to determine the current terminal size.
     """
     if not sys.stdout.isatty():
         return None
+    # FIXME fallback should be integers
```

Comment:
I used the non-integer fallback on purpose: this was the code can know when `get_terminal_size()`  failed and returned the fallback, so the 1000 value can be used